bigbluebutton-tests/playwright/presentation/presentation.js

```diff
@@ -200,6 +200,30 @@ class Presentation extends MultiUsers {
     await frame.waitForSelector(e.ytFrameTitle);
     return frame;
   }
+
+  async presentationFullscreen() {
+    const element = await this.modPage.getLocator(e.presentationContainer);
+    const value = await element.evaluate((e) => {
+      return window.getComputedStyle(e).getPropertyValue("height")
+    });
+    const height = parseInt(value);
+
+    await this.modPage.waitAndClick(e.whiteboardOptionsButton);
+    await this.modPage.waitAndClick(e.presentationFullscreen);
+
+    const element1 = await this.modPage.getLocator(e.presentationContainer);
```

Comment:
Looks like quite a repetition. Even the selector doesn't change (`e.presentationContainer`). You could probably put this into a separate function that returns the height - called something like `getCurrentPresentationHeight`, and call that function twice.

Reply:
Okk, I will create this function to return the height and make those changes, thanks for the tip Maxim!